test/puppetlabs/puppetdb/migration_coordination_test.clj

```diff
@@ -1,19 +1,25 @@
 (ns puppetlabs.puppetdb.migration-coordination-test
   (:require
+   [clojure.java.jdbc :as sql]
    [clojure.test :refer :all]
    [puppetlabs.puppetdb.command.constants :as cmd-consts]
    [puppetlabs.puppetdb.jdbc :as jdbc]
-   [puppetlabs.puppetdb.scf.migrate :refer [desired-schema-version]]
-   [puppetlabs.puppetdb.testutils.db :as tdb :refer [*db*]]
-   [puppetlabs.puppetdb.testutils.services :as svc-utils]
-   [puppetlabs.puppetdb.time :refer [now to-timestamp]]
-   [puppetlabs.puppetdb.testutils.cli :refer [example-report]]))
+   [puppetlabs.puppetdb.scf.migrate :as migrate :refer [desired-schema-version]]
+   [puppetlabs.puppetdb.testutils :refer [default-timeout-ms]]
+   [puppetlabs.puppetdb.testutils.cli :refer [example-report]]
+   [puppetlabs.puppetdb.testutils.db :as tdb
+    :refer [*db* clear-db-for-testing! test-env with-test-db]]
+   [puppetlabs.puppetdb.testutils.services :as svc-utils
+    :refer [call-with-single-quiet-pdb-instance create-temp-config]]
+   [puppetlabs.puppetdb.time :refer [now to-timestamp]])
+  (:import
+   (org.postgresql.util PSQLException)))
 
 (deftest schema-mismatch-causes-new-connections-to-throw-expected-errors
   (doseq [db-upgraded? [true false]]
-    (tdb/with-test-db
-      (svc-utils/call-with-puppetdb-instance
-       (-> (svc-utils/create-temp-config)
+    (with-test-db
+      (call-with-single-quiet-pdb-instance
```

Comment:
I still have to dig into why a bit more, but I think this macro is seeing some of the expected connection errors in these tests and dumping all the logs for the failed connection retries. We may want to go back to `call-with-puppetdb-instance` here and in the test below to suppress the logs or see if we can update `*notable-log-event?*` so that it suppresses the logs for expected errors.